stream-parent/stream-client/src/main/java/org/interledger/stream/sender/AimdCongestionController.java

```diff
@@ -290,6 +289,11 @@ public void setCongestionState(final CongestionState congestionState) {
     return maxPacketAmount;
   }
 
+  @Override
+  public boolean hasInFlight() {
```

Comment:
We need a unit test for this one. @theotherian I seem to recall you wrote this - maybe it didn't get pushed?

Reply:
Nm - I see the test.